client/src/views/home/LoginPage.vue

```diff
@@ -10,7 +10,11 @@
     </div>
     <ion-card class="login-card">
       <ion-card-header class="login-card-header">
-        <organization-card :device="device" />
+        <organization-card
+          :last-login-device="storedDeviceDataDict[device.deviceId]"
```

Comment:
You don't need the `last-login-device` here, you're hiding everything but the org name anyway.